packages/dicomImageLoader/src/shared/decodeImageFrame.ts

```diff
@@ -295,8 +295,18 @@ function _handleTargetBuffer(
     type,
     offset: rawOffset = 0,
     length: rawLength,
+    rows,
   } = options.targetBuffer;
 
+  const TypedArrayConstructor = typedArrayConstructors[type];
+
+  if (!TypedArrayConstructor) {
+    throw new Error(`target array ${type} is not supported`);
+  }
+
+  if (rows && rows != imageFrame.rows) {
+    scaleImageFrame(imageFrame, options.targetBuffer, TypedArrayConstructor);
```

Comment:
I know what this does, but we have another scale (intensity) right below, so we better rename this new one

Reply:
Renamed the intensity scale import, with a comment.